src/h2olog/misc/gen_raw_tracer.py

```diff
@@ -374,7 +374,7 @@ def build_tracer(context, metadata):
   c = r"""// %s
 int %s(struct pt_regs *ctx) {
   const void *buf = NULL;
-  struct h2olog_event_t event = { .id = %s };
+  struct h2olog_event_t event = { .id = %s, .tid = get_h2o_tid(), };
```

Comment:
How about doing `(uint32_t)bpf_get_current_pid_tgid()` here directly?

There's no need to pay the costs of calling another eBPF function, the costs being overhead in terms of execution time, code size, potential interoperability issues (IIRC, you cannot call another eBPF function in older kernels).

Reply:
Adding `inline` is also ok, but in fact there's no need to define an extra utility.  Fixed 22762f1f1